### PR TITLE
Updated Reminders as per Run Book

### PR DESCRIPTION
**Pull request by LeoKHoward**
# Motivation and Context
Business have agreed that rather than setting new 200k batch limit, we can send all of these in one go so removed irrelevant lines as per run book and removed wave three from toolbox

# What has changed
Updated constants
Updated tests

# How to test?
Run tests and ensure it works

# Links
https://trello.com/c/0nt7lbx0
https://collaborate2.ons.gov.uk/confluence/display/SDC/RM+Census+Reminder+Run+Book+-+2021
https://collaborate2.ons.gov.uk/confluence/display/SDC/Census+2021+Scheduled+%28Unbatched%29+Reminder+Action+Rules
https://collaborate2.ons.gov.uk/confluence/display/SDC/Census+2021+Reminder+Daily+Diary

### CODE REVIEW

**Comment by LeoKHoward** [comment]
[Update run book & reminder diary for RP1/2/3 reminders (5)](https://trello.com/c/0nt7lbx0/1475-update-run-book-reminder-diary-for-rp1-2-3-reminders-5)

File: toolbox/reminder_batch_scheduler/constants.py [hunk 1]
```diff
@@ -5,17 +5,14 @@
 """
 
 ACTION_TYPES_FOR_WAVE = {
-    1: ['P_RL_1RL1_1', 'P_RL_1RL2B_1'],
-    2: ['P_RL_2RL1', 'P_RL_2RL2B'],
-    3: ['P_QU_H1', 'P_QU_H2'],
+    1: ['P_RL_1RL1_1', 'P_RL_1RL2B_1', 'P_QU_H1', 'P_QU_H2'],
```
Comment (AdamHawtin) [inline toolbox/reminder_batch_scheduler/constants.py]: I don't think this is right, batch 1 should not include these pack codes.

File: toolbox/tests/reminder_batch_scheduler/test_reminder_batch.py [hunk 2]
```diff
@@ -164,18 +164,16 @@ def test_build_batch_count_query(batch, wave_classifiers, expected_query, expect
         'P_RL_1RL1_1': "case_type != 'HI' AND treatment_code IN ('HH_LP1E', 'HH_LP2E')"
                        " AND survey_launched = 'f' AND print_batch IN ('1','2','3')",
         'P_RL_1RL2B_1': "case_type != 'HI' AND treatment_code IN ('HH_LP1W', 'HH_LP2W')"
-                        " AND survey_launched = 'f' AND print_batch IN ('1','2','3')"
+                        " AND survey_launched = 'f' AND print_batch IN ('1','2','3')",
+        'P_QU_H1': "case_type != 'HI' AND treatment_code IN ('HH_LP1E') AND print_batch IN ('1','2','3')",
+        'P_QU_H2': "case_type != 'HI' AND treatment_code IN ('HH_LP1W') AND print_batch IN ('1','2','3')"
```
Comment (AdamHawtin) [inline toolbox/tests/reminder_batch_scheduler/test_reminder_batch.py]: I don't think this is right, batch 1 should not include these pack codes.

**Review by AdamHawtin — DISMISSED** [review]

**Review by NickGrantX — CHANGES_REQUESTED** [review]
what Adam said

File: toolbox/tests/reminder_batch_scheduler/test_reminder_batch.py (hunk 2 above)
Comment (NickGrantX) [reply toolbox/tests/reminder_batch_scheduler/test_reminder_batch.py]: what does `print_batch` of `3` even mean anymore? It should be meaningless

**Review by NickGrantX — CHANGES_REQUESTED** [review]
```
    'P_QU_H1': "treatment_code IN ('HH_LP1E')",
    'P_QU_H2': "treatment_code IN ('HH_LP1W')"
```

needs to be removed from `ACTION_TYPE_CLASSIFIERS`

**Review by NickGrantX — APPROVED** [review]

**Review by gavinleeedwards — APPROVED** [review]

**Pull request merged**